.changeset/tall-hotels-speak.md

```diff
@@ -0,0 +1,5 @@
+---
+'@vanilla-extract/esbuild-plugin': minor
```

Comment:
Not sure this should be minor or major, actually 🤔

Reply:
This would be a breaking change, so it should be `major` imo

Reply:
This would be a breaking change, so it should be `major` imo

```suggestion
'@vanilla-extract/esbuild-plugin': major
```